build_tools/ci/cpu_comparison/run.py

```diff
@@ -1590,6 +1594,78 @@ def __init__(self):
                 ),
             )
         )
+
+        # Tests Matmul + Trunci with Scaling.
+        # Phoenix : Ukernel + Peano.
+        self.register(
```

Comment:
Not specific to this PR, but I think we should consider factorizing/standardizing tests/functions/templates where possible in this file, it's becoming too large and unwieldy IMO